harvard-lts/CURIOSity
# Upgrading spotlight-oaipmh-resources to 3.0.4.

**Upgrading spotlight-oaipmh-resources to 3.0.4.**
* * *


# What does this Pull Request do?
The new version of the gem allows urns to be case insensitive when harvesting.

# How should this be tested?

A description of what steps someone could take to:
* Rebuild curiosity off of the oaimph-304 branch
* Harvest the items in the ctw-thumb set
* Confirm that all 3 thumbnails show up.


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No
